packages/web/tailwind.config.js

```diff
@@ -139,6 +144,11 @@ module.exports = {
           'blue-60': 'hsl(var(--sb-blue-60))',
           'blue-70': 'hsl(var(--sb-blue-70))',
           'blue-80': 'hsl(var(--sb-blue-80))',
+
+          'purple-10': 'hsl(var(--sb-purple-10))',
```

Comment:
The way I copied these over was super tedious, wondering if I'm missing something, could not find them easily in figma